front/src/components/BoxReconciliationOverlay/components/MatchProductsForm.tsx

```diff
@@ -155,14 +155,18 @@ export function MatchProductsForm({
     <form onSubmit={handleSubmit(onSubmitMatchProductsForm)}>
       <Flex direction="column" gap="2" alignItems="stretch">
         <Text fontSize={16} fontWeight="bold">
-          Product & Gender:{" "}
+          Sender Product & Gender:{" "}
         </Text>
         <Text
           fontSize={16}
-          fontWeight="semibold"
+          fontWeight={productId?.value === "" ? "semibold" : ""}
+          fontStyle="italic"
           style={{ color: productId?.value === "" ? "#FF0000" : "#000" }}
         >
           {shipmentDetail?.sourceProduct?.name}{" "}
+          {shipmentDetail?.sourceProduct?.gender
```

Comment:
Could you handle the none case here @vahidbazzaz ?

![image](https://github.com/boxwise/boxtribute/assets/29065125/db68d5ea-20f7-4070-ba18-3c771fa0f36f)

Reply:
@aerinsol

Reply:
Yes drop the (none) in the no gender case